schemas/groups/environment.json

```diff
@@ -303,27 +303,27 @@
           "units": "rad"
         },
         "directionTrue": {
-          "description": "The wind direction relative to true north",
+          "description": "True Wind Direction (TWD) based on speed through water (STW), relative to true north",
           "$ref": "../definitions.json#/definitions/numberValue",
           "units": "rad"
         },
         "directionMagnetic": {
-          "description": "The wind direction relative to magnetic north",
+          "description": "Wind direction based on speed through water (STW), relative to magnetic north",
           "$ref": "../definitions.json#/definitions/numberValue",
           "units": "rad"
         },
         "speedTrue": {
-          "description": "Wind speed over water (as calculated from speedApparent and vessel's speed through water)",
+          "description": "True Wind Speed (TWS) over water, calculated from apparent wind speed (AWS) and speed through water (STW)",
```

Comment:
sic

Reply:
Ok, will revert to "speedApparent (AWS)" and "vessel's speed over ground"